File: Solutions/1893. Check if All the Integers in a Range Are Covered/1893. Check if All the Integers in a Range Are Covered.java
class Solution {
    public boolean isCovered(int[][] ranges, int left, int right) {
        int[] cnt=new int[51];
        for(var r:ranges)
            for(int x=r[0];x<=r[1];++x)
                cnt[x]=1;
        for(int i=left;i<=right;++i)
            if(cnt[i]==0)
                return false;
        return true;
    }
}

File: Solutions/1893. Check if All the Integers in a Range Are Covered/1893. Check if All the Integers in a Range Are Covered.go
func isCovered(ranges [][]int, left int, right int) bool {
    cnt:=make([]int,51)
    for _,r:=range(ranges){
        for x:=r[0];x<=r[1];x+=1{
            cnt[x]=1
        }
    }
    for i:=left;i<=right;i+=1{
        if cnt[i]==0{
            return false
        }
    }
    return true
}